__tests__/extension-requests/extension-requests.test.js

```diff
@@ -200,7 +200,7 @@ describe.skip('Tests the new Extension Requests Screen', () => {
       const url = interceptedRequest.url();
       if (
         url ===
-        'https://api.realdevsquad.com/extension-requests?size=5&dev=true&order=asc'
+        'https://api.realdevsquad.com/extension-requests?q=size:5,dev:true,order:asc'
```

Comment:
I personally don't think dev=true should go inside query, other filter property are fine

Reply:
made the changes